chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -36,6 +36,9 @@
     libshadow-ruby1.8
     markdown
   )
+  unless search(:node, "platform:windows").empty?
+    pkglist.push "smbclient"
```

Comment:
Not a Ubuntu or Debian user but at least in Debian the name is `samba-client`

https://wiki.debian.org/SambaServerSimple